target/linux/ramips/image/Makefile

```diff
@@ -144,6 +144,11 @@ define Build/uimage-padhdr
 	mv $@.new $@
 endef
 
+define Build/uimage-sgehdr
+	uimage_sgehdr -i $@ -o $@.new  -m $(DEVICE_MODEL) -h $(DEVICE_VARIANT) -s V1.00000
```

Comment:
Nitpick: two spaces after $@.new ...

Reply:
Probably introduced when removing the line wrap that should be here.

Reply:
removed space & added line wrap

Reply:
@adschm I guess the whitespace/line wrap issue is fixed now?